src/main/java/graphql/schema/CoercingEnvironment.java

```diff
@@ -0,0 +1,65 @@
+package graphql.schema;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Locale;
+
+/**
+ * This interface represent the parameters that are sent into a {@link Coercing} methods
+ *
+ * @param <T> for two
+ */
+public interface CoercingEnvironment<T> {
```

Comment:
Feedback - kill it - its not an overhead we want to add in terms of object allocation